README.md

```diff
@@ -1,370 +1,173 @@
 ﻿![Azure Functions Logo](https://raw.githubusercontent.com/Azure/azure-functions-core-tools/refs/heads/main/eng/res/functions.png)
 
 |Branch|Status|
-|---|---|
-|main|[![Build status](https://azfunc.visualstudio.com/Azure%20Functions/_apis/build/status/azure-functions-core-tools?branchName=main)](https://azfunc.visualstudio.com/Azure%20Functions/_build/latest?definitionId=11&branchName=main)|
-|in-proc|[![Build status](https://azfunc.visualstudio.com/Azure%20Functions/_apis/build/status/azure-functions-core-tools?branchName=in-proc)](https://azfunc.visualstudio.com/Azure%20Functions/_build/latest?definitionId=11&branchName=in-proc)|
-|v1.x|[![Build status](https://azfunc.visualstudio.com/Azure%20Functions/_apis/build/status/azure-functions-core-tools?branchName=v1.x)](https://azfunc.visualstudio.com/Azure%20Functions/_build/latest?definitionId=11&branchName=v1.x)|
+|------|------|
+|main|[![Build Status](https://dev.azure.com/azfunc/public/_apis/build/status%2Fazure%2Fazure-functions-core-tools%2Fcoretools.public?repoName=Azure%2Fazure-functions-core-tools&branchName=main)](https://dev.azure.com/azfunc/public/_build/latest?definitionId=579&repoName=Azure%2Fazure-functions-core-tools&branchName=main) |
+|in-proc|[![Build Status](https://dev.azure.com/azfunc/public/_apis/build/status%2Fazure%2Fazure-functions-core-tools%2Fcoretools.public?repoName=Azure%2Fazure-functions-core-tools&branchName=in-proc)](https://dev.azure.com/azfunc/public/_build/latest?definitionId=579&repoName=Azure%2Fazure-functions-core-tools&branchName=in-proc)|
 
 # Azure Functions Core Tools
 
 The Azure Functions Core Tools provide a local development experience for creating, developing, testing, running, and debugging Azure Functions.
 
-## Versions
-
-**v1** (v1.x branch): Requires .NET 4.7.1 Windows Only
-
-**v4**: (main branch): Self-contained cross-platform package **(recommended)**
-
-## Installing
-
-### Windows
-
-#### To download and install with MSI:
-
-##### v4
-
-- [Windows 64-bit](https://go.microsoft.com/fwlink/?linkid=2174087) (VS Code debugging requires 64-bit)
-- [Windows 32-bit](https://go.microsoft.com/fwlink/?linkid=2174159)
-
-##### v3
-
-- [Windows 64-bit](https://go.microsoft.com/fwlink/?linkid=2135274) (VS Code debugging requires 64-bit)
-- [Windows 32-bit](https://go.microsoft.com/fwlink/?linkid=2135275)
-
-#### To install with npm:
-
-##### v4
-```bash
-npm i -g azure-functions-core-tools@4 --unsafe-perm true
-```
+## Usage
 
-##### v3
-```bash
-npm i -g azure-functions-core-tools@3 --unsafe-perm true
-```
+To use the Azure Functions Core Tools, open a terminal and run the `func` command. You can see the available commands and options by running:
```

Comment:
For the first time users, we should say to install it first. We can start the sentence with "Once installed" or "Post installation".